init.lua

```diff
@@ -1,4 +1,5 @@
 require("plugin_manager")
+require("keymaps")
```

Comment:
For now maybe keep the keymaps inline at the end of `init.lua` just before the sourcing of  user config?
If it grows too large we can split into modules maybe.

Reply:
✅